playbooks/unmount-and-clear-disks.yml

```diff
@@ -0,0 +1,30 @@
+
+---
+- name: unmount and clear all drives
+  hosts: "{{ hosts }}"
+
+  tasks:
+  - name: install dependencies
+    apt: pkg="gdisk"
+
+  - set_fact: disk_list={{ swift.disks }}
+    when: hosts == "swiftnode"
```

Comment:
Why are you setting a fact for a variable that already exists? I don't see any else that will set `disk_list`, so why not just use `swift.disks` everywhere?